index.js

```diff
@@ -76,15 +77,23 @@ Ultron.prototype.remove = function remove() {
   if (args.length === 1 && 'string' === typeof args[0]) {
     args = args[0].split(/[, ]+/);
   } else if (!args.length) {
-    args = [];
+    if (ee.eventNames) {
+      args = ee.eventNames();
+    } else if (ee._events) {
+      args = [];
 
-    for (event in this.ee._events) {
-      if (has.call(this.ee._events, event)) args.push(event);
+      for (event in ee._events) {
+        if (has.call(ee._events, event)) args.push(event);
```

Comment:
It is possible that we are in a browsert that doesn't have `Object.create` support and this is an instance of `EventEmitter3` < 1.2.0.
In this case we fill the array with prefixed event names, and this will obviously fail below when we get the listeners via `ee.listeners(name)` as our name will be prefixed again.
The ideal solution would be to detect if this is an instance of `EventEmitter3` and in that case see if it is prefixed and slice the event name accordingly.

The problem here is that in order to do this cleanly we need to add a new method to `EventEmitter3` and publish it as `1.1.2`. Another option would be something like this:
```js
var EE3;
try {
  EE3 = require('eventemitter3');
} catch(e) {};
```
and check if `ee` is an instance of `EE3` but this seems fragile.

Do you have better ideas?